### ISSUE TITLE
How to add html text to label text?

### ISSUE THREAD

**Issue by KoNGoL** [almende/vis#1936]
I need to modify the label text so I can change the color for each line of the text. It is possible to achive? Adding some html or css text?

**Comment by mojoaxel** [comment]
Please have a look at the [customCss Example](/almende/vis/blob/master/examples/timeline/styling/customCss.html). Here the content includes html tags and css.

To change the color you could use css or something like this:
```html
<ul>
  <li style="color:black;">line1</li>
  <li style="color:red;">line2</li>
  <li style="color:yellow;">line3</li>
</ul>
```

**Comment by KoNGoL** [comment]
It doesn't work for me. This is what I try.
![vis-html-example](https://cloud.githubusercontent.com/assets/20227402/16869640/12f6c652-4a7e-11e6-9407-cd664db46317.png)

And this is the vis options.
```
var optionsFA = {
                autoResize: true,
                height: '500px',
                layout: {
                    hierarchical: {
                        sortMethod: 'directed',
                        direction: 'RL',
                        levelSeparation: 250
                    }
                },
                nodes: {
                    shadow:true,
                    font: '12px arial black'
                },

                edges: {
                    width: 2,
                    shadow:true,
                    color: '#7f92a3',
                    smooth: {enabled: true,
                        roundness: 0.4}
                },

                groups: {
                    industry: {
                        shape: 'icon',
                        icon: {
                            face: 'FontAwesome',
                            code: '\uf275',
                            size: 50,
                            color: '#1e5eaf'
                        }
                    },
                    users: {
                        shape: 'icon',
                        icon: {
                            face: 'FontAwesome',
                            code: '\uf0c0',
                            size: 40,
                            color: '#1e5eaf'
                        }
                    },
                    supplier1: {
                        shape: 'icon',
                        icon: {
                            face: 'FontAwesome',
                            code: '\uf275',
                            size: 40,
                            color: '#1e5eaf'
                        }
                    },
                    supplier2: {
                        shape: 'icon',
                        icon: {
                            face: 'FontAwesome',
                            code: '\uf275',
                            size: 30,
                            color: '#1e5eaf'
                        }
                    },
                    supplier3: {
                        shape: 'icon',
                        icon: {
                            face: 'FontAwesome',
                            code: '\uf275',
                            size: 20,
                            color: '#1e5eaf'
                        }
                    }
                }
            };
```

And the nodes creation.

```
nodes.push({
                    id: company.IdEmpresa,
                    label: "<p>" + company.Nombre + "<br/>Inventario: " + currentInvetory + "</p>",
                    group: company.IdCliente == 0? "industry": "supplier1",
                    name: company.Nombre,
});
network = new vis.Network($('#networkContainer')[0], dataFA, optionsFA);
```

Any suggestions?

**Comment by mojoaxel** [comment]
It looks like HTML-labels are not possible for network charts at the moment.

Simple multiline labels are possible: [multilineText](http://visjs.org/examples/network/labels/multilineText.html) and [disassemblerExample](http://visjs.org/examples/network/exampleApplications/disassemblerExample.html), but I could not find a way of changing colors inside a label. 

I created a new Feature Request for that: #1965

**Comment by mojoaxel** [comment]
@KoNGoL Can we close this issue? CSS in Labels is just not possible at the moment.

**Comment by mojoaxel** [comment]
implemented by @eymiha with #2385

**Comment by shoabshah** [comment]
> It looks like HTML-labels are not possible for network charts at the moment.
> 
> Simple multiline labels are possible: [multilineText](http://visjs.org/examples/network/labels/multilineText.html) and [disassemblerExample](http://visjs.org/examples/network/exampleApplications/disassemblerExample.html), but I could not find a way of changing colors inside a label.
> 
> I created a new Feature Request for that: #1965

Is it Sill (Apr - 2019) not possible??
